doc/ref_cert/RC2/chapters/chapter03.md

```diff
@@ -2,3 +2,89 @@
 
 # 3. Kubernetes Test Cases and Traceability to CNTT Requirements
```

Comment:
Kubernetes based infrastructure Test Cases and Traceability to Anuket Requirements